app/models/chargeback_rate_detail.rb

```diff
@@ -92,4 +92,22 @@ def rate_type
     # Return parent's rate type
     chargeback_rate.rate_type unless chargeback_rate.nil?
   end
+
+  # New method created in order to show the rates in a easier to understand way
+  def show_rates
+    rate = self.rate.to_s
+    return detail_currency.code if rate.to_f.zero?
+    per_time_map = {
+      :hourly  =>"Hour",
+      :daily   => "Day",
+      :weekly  => "Week",
+      :monthly => "Month",
+      :yearly  => "Year"
+    }
+    hr = per_time_map[per_time.to_sym]
+    raise "rate time unit of '#{per_time}' not supported" if hr.nil?
+    rate_display = "#{detail_currency.code} / #{hr}"
+    rate_display_unit = "#{rate_display} / #{per_unit_display}"
+    per_unit.nil? ? rate_display : rate_display_unit
```

Comment:
This method is handling exceptional cases and building strings for presentation in the UI.  Perhaps we can extract the presentation layer out.  @martinpovolny do you have suggestions?

Please add a series of tests for the conditionals this method is checking.